src/components/FileCard/FileCard.vue

```diff
@@ -11,8 +11,8 @@
                 @click="emit('onClickImage')"
             />
             <!-- Favorite -->
-            <div class="file-card__stacked button left" @click="emit('onClickFavorite')">
-                <i class="star icon large" :class="{ outline: !isFavorite ? 'outline' : '' }" />
+            <div v-if="canbeMarkedAsFavorite" class="file-card__stacked button left" @click="emit('onClickFavorite')">
```

Comment:
Could we change canBeMarkedAsFavorite to favoritable or hideFavorite instead?
Thinking that it would be quite long in template part like

`<FileCard can-be-marked-as-favorite>`
vs
`<FileCard favoritable>`
`<FileCard hide-favorite>`

Reply:
Changed to hide-favorite